profiling-ffi/src/exporter.rs

```diff
@@ -211,7 +232,7 @@ pub unsafe extern "C" fn ddog_prof_Exporter_Request_build(
 ///
 /// # Arguments
 /// * `exporter` - borrows the exporter for sending the request
-/// * `request` - takes ownership of the request
+/// * `request` - takes ownership of the request. Be careful not to drop it twice!
```

Comment:
Minor: I suggest being even more explicit here:

```suggestion
/// * `request` - takes ownership of the request. You do not need to (and must not) call `Request_drop` on a request after calling `send` with it.
```